README.md

```diff
@@ -29,12 +22,30 @@ Open terminal and go to root directory of the project and run:
 pytest
 ````
 
+## Configuration
+
+The properties in data_service/config/.env are meant for Bucket configuration 
+and are applicable when the service is deployed within a Docker container.
```

Comment:
Properties in .env file are applied when you run application from everywhere, i.e. IDE, command line, Docker container.